mernarezk/assignment1/Q2.java

```diff
@@ -0,0 +1,37 @@
+package Assignment1;
+
+public class Q2 {
+	/**
+	 * Method that gets Kth node from back of a singly linkedlist.
+	 * @param head of the linkedlist
+	 * @param kth node to be returned.
+	 */
+	public ListNode getKth(ListNode head, int k) {

```

Comment:
optional: Please also add a few test calls to this method.

Reply:
Hey Izabela! I added JUnit tests for Q1 but I am skipping Q2 because it will need lots of code changes and I am kind of late in submitting my review! Sorry Had my masters projects discussions ^_^